tests/foreman/smoke/test_ui_smoke.py

```diff
@@ -475,8 +472,8 @@ def test_puppet_install(self):
                 vm.register_contenthost(activation_key_name, org_name)
                 vm.configure_puppet(rhel6_repo)
                 host = vm.hostname
-                session.nav.go_to_hosts()
                 set_context(session, org=ANY_CONTEXT['org'])
+                session.nav.go_to_hosts()
```

Comment:
According to screenshots from CI and performed investigation - that approach should help as it seems that selenium clicks on 'Manage Location' link instead selecting necessary checkbox from the list as corresponding dropdown (with locations and organization) is still visible after set_context operation. That issue should be investigated more and is out of scope of current PR